geth/notification/fcm/notification.go

```diff
@@ -0,0 +1,55 @@
+package fcm
+
+import (
+	"github.com/NaySoftware/go-fcm"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/notification"
+)
+
+// Notification represents messaging provider for notifications.
+type Notification struct {
```

Comment:
`Client` would be easier to reason about. Seeing `Notification` type, I imagine a struct with some fields that is an actual notification, not something that sends notifications.